src/components/WebexLocalMedia/WebexLocalMedia.js

```diff
@@ -0,0 +1,32 @@
+import React, {Fragment} from 'react';
+import PropTypes from 'prop-types';
+
+import {WebexAvatar} from '../';
+import {useMeeting, useVideo} from '../hooks';
+
+import './WebexLocalMedia.scss';
+
+/**
+ * Webex Local Media component displays the user's local video
+ * previous to joining a meeting from a given meetingID.
+ *
+ * @param {object} props
+ * @returns {object} JSX of the component
+ */
+export default function WebexLocalMedia({meetingID, personID}) {
+  const {localVideo} = useMeeting(meetingID);
+  const videoRef = useVideo(localVideo);
+  const localMedia = (
+    <Fragment>
+      <audio playsInline />
```

Comment:
Local media shouldn't have audio. This will only be needed on the remote stream.

Reply:
ahhh, I was a bit confused about this too.